### DISCUSSION TITLE
Video final

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#179] [Ideas]
El proyecto final tiene que incluir una demo de 10 minutos donde todos hablemos.

![image](https://user-images.githubusercontent.com/15602473/196852103-1d7d9286-15c6-49f9-a847-8d9335fe30d5.png)

Nuestro sitio posee suficientes páginas para que todos hablen, sin siquiera contar todo lo de back-end, así que cada uno puede incluso tomar un par de páginas en las que haya ayudado mucho y más o menos quiera exponer.

- Página principal (más la idea de que la información que ahí aparece viene de la creación de la subasta por parte del martillero)
- Quiénes somos
- Cómo funciona
- Catálogo
- Contacto anónimo
- Identificarse (login)
- Registración
- Reestablecer contraseña

Después las específicas que se acceden directamente por el disco porque no podemos navegarlas:
- Contacto para miembros (misma página para pujador y consignatario)
- Subasta (pujador)
- Mis compras (pujador)
- Perfil (misma página para pujador, consignatario, martillero)
- Consignaciones (consignatario)
- Subasta (martillero)
- Crear consignatario (martillero)
- Crear artículo (martillero)
- Crear subasta y elegir lotes (martillero)

Además hay que hablar de la parte de back-end, sin embargo no dice que todos tienen que hablar de todo porque supongo que sería muy caótico así que la parte de back-end la puedo resumir yo.

Como tenemos solo 10 minutos lo mejor es que cada uno practique lo que quiere decir sobre el proyecto y que más o menos calcule entre un minuto y un minuto y medio de tiempo. Tienen que hablar de por qué existe/n esa/s página/s, cómo está creada, si usa bootstrap o si usa css simple, si usa javascript, etc. No es obligatorio hablar mucho, si no se les ocurre mucho agreguen una página más a lo que van a explicar o corten ahí, Lo importante es que no hablen 30 segundos cada uno y luego me dejen a mí 5 minutos para rellenar :-P

El sprint termina el lunes 14 así que la idea es hacer la grabación el 13 o el 6, una semana antes, domingo durante la reunión de sprint en donde estamos todos y podemos hacer varias tomas, Supongo que se puede editar así que al final podemos cortar los diálogos y juntar todo en un video en caso de ser necesario (habría que averiguar).

**Comment by AhumadaCeleste** [comment]
En función de lo indicado tomo en principio:
**-Subasta (martillero)
-Crear artículo (martillero)
-Reestablecer contraseña.**
Esta cantidad considerando que observo que somos 7 participando. En tal caso podríamos hacer más o menos.

**Comment by NataliaAlvarezIspc** [comment]
Buenas!
Hablaría de:

- Login
- Registración.
- Index.

Podría también incluir algo del Script de BD???
Capaz que podemos conversarlo en la reunión de este domingo para acordar que diríamos.

**Reply by rpgrca** [reply]
Sí, hablá de lo que quieras pero más o menos vamos a tener entre un minuto y dos minutos cada uno. Vamos a grabarlo por partes y después juntamos todo en un solo video. Yo creo que @AhumadaCeleste con crear artículo y subasta tiene mucho y que te podría pasar la de reestablecer contraseña a vos para que una sola persona hable de toda esa página completa.

**Comment by AhumadaCeleste** [comment]
Me quedo entonces "subasta martillero", y "acerca de" que es cortita de texto si les parece bien, y muestro un poco de la maquetación en papel que hice.

**Reply by rpgrca** [reply]
No hay problema!

**Comment by claubott** [comment]
Hola! Tengo una sugerencia de los temas a desarrollar en el video:

1. presentación del proyecto

2. documentacion
-ieee830

3. front end
-tecnologias usadas
-funcionalidad/demo
   • vistas generales
   • vistas usuarios registrados (pujadores/consignatarios)
   • vistas martillero

4. BD
-diseño
-implementacion

5. backend
-fastapi/justificación

La demo de las vistas se puede dividir entre 3 personas, y todo esto daría para que hablemos las 7 personas que estamos todavía con el proyecto.

**Reply by rpgrca** [reply]
Yo saltearía la parte de la documentación porque digamos que es lo que conocen y ya corrigieron. La parte de front-end se puede dividir entre varios, sí, de hecho ya tenemos cubierto la parte de registración, login y reestablecer contraseña, también la vista del martillero con respecto a la subasta.

Podríamos dividir el resto de manera lógica que podría ocupar a tres o cuatro personas:
- Por un lado parte de pujador que incluiría la opción de compras y subasta,
- Por otro lado todo lo relacionado a los consignatarios (incluyendo la parte del martillero para crear consignatarios, la parte del martillero para crear artículos del consignatario y la parte del consignatario para ver sus consignaciones).
- Después alguien podría tomar la página principal y el catálogo, además de la parte del martillero para crear la subasta.
- Y tenemos la parte de contacto también pero que tiene el problema de que no tenemos ningún lugar donde se recibirían los mensajes en el sistema así que habría que justificar por qué decidimos que desde el contacto se envía un mail al martillero y que el martillero responda desde su cuenta de mail a los mensajes en lugar de tenerlos en una página.

Como Natalia ya tomó el front-end yo tomaría toda la parte de back-end salvo que alguien quiera estudiarlo. La grabación la vamos a hacer el domingo 6 de noviembre a la mañana y a la noche los que no puedan así que las cosas que nadie tome para ese día las tomaremos con el resto o las dejaremos de lado.

**Comment by Brian-Almada** [comment]
![2022-11-03_16h35_46](https://user-images.githubusercontent.com/100881466/199817604-488639d8-6afa-40e1-80ae-f4e4ea2c57ca.png)
Tomo esa parte para el video!

**Comment by claubott** [comment]
Yo tomo los 2 formularios de contacto y el perfil. Agrego en modo tabla qué ha tomado cada uno:

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 14">
<link id=Main-File rel=Main-File
href="file:///C:\Users\cb\AppData\Local\Temp\msohtmlclip1\01\clip.htm">
<link rel=File-List
href="file:///C:\Users\cb\AppData\Local\Temp\msohtmlclip1\01\clip_filelist.xml">
<style>

</style>
</head>

<body link=blue vlink=purple>



  | generales
-- | --
  |  
brian | Página principal (hay info q viene de subasta martillero)
celeste | Quiénes somos
  | Cómo funciona
brian | Catálogo
claudia | Contacto anónimo
natalia | Identificarse (login)
natalia | Registración
natalia | Reestablecer contraseña
  |  
  | específicas
  |  
claudia | Contacto para miembros (misma página para pujador y consignatario)
  | Subasta (pujador)
  | Mis compras (pujador)
claudia | Perfil (misma página para pujador, consignatario, martillero)
  | Consignaciones (consignatario)
celeste | Subasta (martillero)
  | Crear consignatario (martillero)
  | Crear artículo (martillero)
  | Crear subasta y elegir lotes (martillero)
  |  
roberto | backend



</body>

</html>

**Comment by rocappellano** [comment]
¡Hola! Me asigno la de 
Cómo funciona
Subasta
Mis Compras
¡Si les parece bien!
